samples/snippets/predict_tabular_forecasting_sample.py

```diff
@@ -0,0 +1,44 @@
+# Copyright 2020 Google LLC
```

Comment:
Same as corresponding classification sample.

Reply:
Is it possible that all the predict tabular samples would be identical?  Is so we should consolidate those too perhaps. On the other hand if the sample tests would be very different (Eg you might be looking for different specific fields in the predictions), then let’s keep this.

Reply:
Predict cannot be used with 'tabular'. They only support `batch_predict`, I will think about consolidation when doing batch_predict next week.